bcda/cclf/cclf_test.go

```diff
@@ -351,6 +354,47 @@ func (s *CCLFTestSuite) TestGetPriorityACOs() {
 	}
 }
 
+func (s *CCLFTestSuite) TestImportRunoutCCLF() {
+	db := database.GetGORMDbConnection()
+	defer database.Close(db)
+
+	cmsID := "RNOUT"
+	defer func() {
+		s.NoError(deleteFilesByACO(cmsID, db))
+	}()
+
+	tests := []struct {
+		name     string
+		fileType models.CCLFFileType
+	}{
+		{"Default file type", models.FileTypeDefault},
+		{"Runout file type", models.FileTypeRunout},
+	}
+
+	for _, tt := range tests {
+		s.T().Run(tt.name, func(t *testing.T) {
+			fileName, cclfName := createTemporaryZipFile(s.T())
+			defer os.Remove(fileName)
+
+			metadata := &cclfFileMetadata{
+				cclfNum:   8,
+				name:      cclfName,
+				acoID:     cmsID,
+				timestamp: time.Now(),
+				perfYear:  20,
+				fileType:  tt.fileType,
+				filePath:  fileName,
+			}
+
+			s.NoError(importCCLF(context.Background(), metadata, noopImporter{}))
```

Comment:
Using noopImporter because we need to validate the the fileType is being saved as expected.